idaes/apps/caprese/examples/cstr_rodrigo/cstr_rodrigo_model2.py

```diff
@@ -0,0 +1,174 @@
+#################################################################################

```

Comment:
Why is this file titled `cstr_rodrigo_model2`? If there is no other "CSTR Rodrigo" model, we should probably omit the number.

Reply:
Yeah, I will only show this model so it's better to omit the number.